consumer-rest-core/src/main/scala/za/co/absa/spline/consumer/rest/controller/ErrorControllerAdvice.scala

```diff
@@ -33,9 +34,10 @@ class ErrorControllerAdvice {
   def handle_404 = new ResponseEntity(NOT_FOUND)
 
   @ExceptionHandler(Array(
-    classOf[MethodArgumentTypeMismatchException]
+    classOf[TypeMismatchException],
+    classOf[HttpMessageConversionException]
   ))
-  def handle_400(e: MethodArgumentTypeMismatchException) = new ResponseEntity(ErrorMsg(e.getMessage), BAD_REQUEST)
+  def handle_400(e: Exception) = new ResponseEntity(ErrorMsg(e.getMessage), BAD_REQUEST)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method name does not match the regular expression '^([a-z][A-Za-z0-9]*(?:_=)?|<<|>>>|>|==|!=|<|<=|>|>=|\||&|\^|\+|-|\*|\/|%|\|\||&&|\+\+|--|\+=|-=)$'](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4536902)